SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestCrossServerRPC/CrossServerRPCCube.cpp

```diff
@@ -1,23 +1,50 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 
 #include "CrossServerRPCCube.h"
+#include "EngineClasses/SpatialNetDriver.h"
+#include "EngineClasses/SpatialPackageMapClient.h"
 #include "Net/UnrealNetwork.h"
 
 ACrossServerRPCCube::ACrossServerRPCCube()
```

Comment:
Is this actor only used in this test case? If it's used elsewhere, worth creating a NonReplicatingCrossServerRPCCube

Reply:
As it is used elsewhere I have created a NonReplicatedCrossServerRPCCube.